docs/README.md

```diff
@@ -123,3 +130,420 @@ something like
+>
+> - `boost` is the Booster CLI
+> - `new:<resource>` new is a CLI command, :project tells booster the kind of resource
+> - `boosted-blog` is a parameter for the `new:project` command
+> project name
+
+The `new:project` command generates some scaffolding for you. The project name will be the
+project's root so `cd` to it:
+
+```shell
+cd boosted-blog
+```
+
+There you should have these files and directories already generated:
+
+```text
+boosted-blog
+├── .eslintignore
+├── .gitignore
+├── package-lock.json
+├── package.json
+├── src
+│   ├── commands
+│   ├── common
+│   ├── config
+│   │   └── config.ts
+│   ├── entities
+│   ├── events
+│   └── index.ts
+└── tsconfig.json
+```
+
+Now open the project in your most preferred IDE, e.g. [Visual Studio Code](https://code.visualstudio.com/).
+
+#### 2. First command
+
+We will now define our first [command](#commands), it will allow us to create posts in our
+blog. Write this command in the project's root directory:
+
+```bash
+boost new:command CreatePost --fields postId:UUID title:string content:string author:string
+```
+
+The `new:command` creates most of the code for us, take a look at it here:
+
+```text
+boosted-blog
+└── src
+    └── commands
+        └── CreatePost.ts
+```
+
+We still need to define two more things:
+
+1. Who is authorized to run this command.
+1. And what events are being triggered when the command is executed.
+
+We will create the first Event later on this guide so, let's start authorizing `'all'` to
+run this command, to do so edit `src/commands/CreatePost.ts` like this:
+
+```typescript
+@Command({
+  authorize: 'all', // Specify authorized roles here. Use 'all' to authorize anyone
+})
+export class CreatePost {
+  public constructor(
+    readonly postId: UUID,
+    readonly title: string,
+    readonly content: string,
+    readonly author: string
+  ) {}
+
+  public handle(register: Register): void {
+    register.events(/* YOUR EVENT HERE */)
+  }
+}
+
+```
+
+#### 3. First event
+
+It makes sense to emit an event after executing the command `CreatePost`, something like
+`PostCreated`. Let's do it by running this command:
+
+```bash
+boost new:event PostCreated --fields postId:UUID title:string content:string author:string
+```
+
+The `new:event` command generates a new file under the `src/events` directory. The name of
+the file is the name of the Event:
+
+```text
+boosted-blog
+└── src
+    └── events
+        └── CreatePost.ts
+```
+
+Edit the `entityID` method to return the id of the entity this Event is referring to. It
+should look like this:
+
+```typescript
+// src/events/PostCreated.ts
+
+@Event
+export class PostCreated {
+  public constructor(
+    readonly postId: UUID,
+    readonly title: string,
+    readonly content: string,
+    readonly author: string
+  ) {}
+
+  public entityID(): UUID {
+    return this.postId;
+  }
+}
+```
+
+Now that we have an event, we can make the `CreatePost` command to emit it. Let's change
```

Comment:
I'm not sure about the use of `make` here. What do you think?

Reply:
I'm not really sure because [make is super-polysemic](https://www.merriam-webster.com/dictionary/make). In that sentence it sounds to me like _now we can make this thing to happen_ but I'll change it by `edit` because it keeps the sense and we avoid using make.